recipes/visceral-evaluatesegmentation/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: visceral-evaluatesegmentation
+  version: "2015.07.02"
+
+source:
+  fn: EvaluateSegmentation-2015.07.02-Ubuntu.zip # [linux]
+  url: https://github.com/Visceral-Project/EvaluateSegmentation/raw/master/builds/Ubuntu/EvaluateSegmentation-2015.07.02-Ubuntu.zip # [linux]
```

Comment:
Had no time yet to build it from scratch. Sadly it is not documented and linking to an unknown ITK can be tricky.